cmd/agentctl/cli/config.go

```diff
@@ -0,0 +1,65 @@
+//  Copyright (c) 2019 Cisco and/or its affiliates.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at:
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
+package cli
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/ligato/cn-infra/config"
+	"github.com/ligato/cn-infra/logging"
+)
+
+const (
+	configFileDir  = ".agentctl"
+	configFileName = "all.conf"
```

Comment:
Could the default config file name be rather named: `config.yml` to make it clear to user what the **file contains** (configuration) and which **data-serialization format** is used for its contents (YAML).

Reply:
it was waiting for someone to suggest better name, thanks, I'll change it